File: Python/ReverseBitsTest.py
from unittest import TestCase

from ReverseBits import ReverseBits


class TestReverseBits(TestCase):
    def test_reverseBits(self):
        rb = ReverseBits()

        self.assertTrue(0 == rb.reverseBits(0))

        self.assertTrue(43261596 == rb.reverseBits(964176192))

File: Golang/ReverseBits/ReverseBits_test.go
package ReverseBits

import "testing"

func TestReverseBits(t *testing.T) {

	if reverseBits(0) == 0 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if reverseBits(964176192) == 43261596 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}
